src/components/forms/create-survey-form.tsx

```diff
@@ -12,9 +14,29 @@ import { Input } from "@/components/ui/input";
 import { Button } from "../ui/button";
 import { Label } from "../ui/label";
 import { PlusIcon } from "lucide-react";
-import { Slider } from "../ui/slider";
+import createSurveyData from "@/utils/surveyApiClient";
+import { useState } from "react";
 
 export function CreateSurveyDialog() {
+  const [name, setName] = useState<string>(""); // Zustand für den Namen
+  const [userId] = useState<string>(""); // Zustand für den Benutzer-ID
```

Comment:
The API should read the userid from the request. We don't need it here in the frontend